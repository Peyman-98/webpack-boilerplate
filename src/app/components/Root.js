// @flow

import React from 'react'

export default function Root (
  {message = 'Hello world'}: {message: string}
) {
  return <h1>{message}</h1>
}
