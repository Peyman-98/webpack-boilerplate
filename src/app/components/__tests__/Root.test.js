import React from 'react'
import renderer from 'react-test-renderer'
import Root from '../Root'

describe('Root', () => {
  it('render custom message', () => {
    const component = renderer.create(<Root message='yeahh' />)

    expect(component).toMatchSnapshot()
  })
})
