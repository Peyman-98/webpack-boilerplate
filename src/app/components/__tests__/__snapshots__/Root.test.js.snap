// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Root render custom message 1`] = `
<h1>
  yeahh
</h1>
`;
